libstd/src/std/fmt.inko

```diff
@@ -69,3 +69,11 @@ trait pub Format {
   # Formats `self` in a human-readable format for debugging purposes.
   fn pub fmt(formatter: mut Formatter)
 }
+
+# Convenience method to format a value using the DefaultFormatter.
+fn pub fmt(value: ref Format) -> String {
```

Comment:
While we don't have a tool to generate source code documentation yet, adding an example would be nice (so we don't have to catch-up with this in the future).
This method should also be generic, so in the future we can specialise it if necessary (instead of going through dynamic dispatch):

```inko
# Formats a value using the default formatter.
# 
# # Examples
#
#     import std::fmt::(fmt)
#
#     fmt([10, 20]) # => '[10, 20]'
fn pub fmt[T: Format](value: ref T) -> String {
  ...
}
```

Generic vs dynamic dispatch doesn't matter much now, but in the future we'll hopefully better optimise generic methods.